server/bookingValidate.js

```diff
@@ -0,0 +1,85 @@
+import { body } from "express-validator";
+
+const validate = [
+	body("name")
+		.exists({ checkFalsy: true })
+		.isLength({ min: 2 })
+		.not()
+		.isNumeric(),
+	body("surname")
+		.exists({ checkFalsy: true })
+		.isLength({ min: 2 })
+		.not()
+		.isNumeric(),
+	body("age").isNumeric().isLength({ max: 2 }),
```

Comment:
This code is well structured and easy to understand. Great work on validation on each field.